TradeMessage.capnp

```diff
@@ -452,6 +453,13 @@ struct SymbolProperties{
 }
 
 
+struct ExchangeLimits{
+    rateLimit @0 :UInt64;                   # max number of requests per second
```

Comment:
what would be returned for Kraken? They have no limits on place, replace, cancel orders, but they do on account balances, positions, etc?